### PR TITLE
Signal transmission in progress. Prepare linguistic analyst.

### PR DESCRIPTION
**Pull request by I-am-Erk**
#### Summary
Content "Adds basic form of exodii trader"

#### Purpose of change
~~Continue adding the Exodii faction, a non-hostile trader faction of cyborgs.~~
Carry on pipin' in the Exodii folks an' large, them as made of flesh an' metal, tassed to trade as it be. 

#### Describe the solution
~~Adds "Rubik", an almost-English-speaking merchant acting as a front 'borg for the local exodii node. They will tell you who they are, what they're doing here, and how to trade with them, and will make trades and arrange for you to be CBM'd up if you like.~~
Us're apped as Rubik, kennit, an' in this otherland us're tassed to trade with you, for you yarks sommat close to the King's Anglic. 

~~This addition will add Rubik's dialogue, a mutation that gives them a unique sprite, the option to get your first CBMs and purchase and install more, and at least a first pass on their shop items. It will not add missions, or at least not more than a cursory first mission to get the infrastructure in place. If I am lucky I will also get to the ability to develop trust with Rubik and unlock more CBMs. I do not think I will get any more advanced exodii mechanics in place on this pass, that is plenty.~~
For an' all to bring what us has to say, and what us are shinin' like to a bright eye, an I'm ken - that is to say, how us are lookin'. That, an' should you ask us with a rank 'n file, us'll help you with your own flesh an' bone. Now's no time for much to tass for, though, I'd razz. Barton's word, if you come an' yark an' spend your scrip with ol' Rubik, us'll form a fast friend. That one I'm ken for cert.

~~This PR will not add Rubik to the map, unless we get the exodii base done first.~~
Us'll stay nicely-like in JSON for a spell, for an' our home be in the between-lands yet.

~~As with many of my PR's, I'm going to keep Rubik's data all in one or two files for easy testing until this is ready.~~
For this'n and you'll have to ask as the Great Grey, for us aren't cert what it means.

#### Describe alternatives you've considered
~~The universe is one of infinite possibilities.~~
An' it's all an alternate, I'd razz. Every tap an' peck.

#### Testing
~~WIP~~
That'n's havin' to wait for it's done, ain't it?

#### Additional context
~~Note to self: I'm gonna need to add some context cues for translators with `"//~"`~~
Them as ain't with the proper Anglic, they's to need some o' the ol' clarity-draught, if they's to kennit.

~~Some errors in exodii base needed to be fixed up.~~
This'n's allas tassed wi' cleanin' up our neckawoods after we wibble-wobbled in, kennit?

### CODE REVIEW

**Comment by Maleclypse** [comment]
Love the NPC text so far!

File: data/json/npcs/exodii/exodii_merchant_Rubik.jsonbackup [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+      "no": "Ah, well.  Us'll put that into the key and lock, if'n you're ken.  Mayhap you could bring us a mite and trade, and we'll see if we can fix you good soon and soon.  Let no exodii turn away a willin' Benny, and so it is."
+    },
+    "//~": "English: Yes: Oh, well.  Now you're speaking my language.  We can fix you up good, if you're willing to pay us back.  I'll say it as clearly as I can.  If you help us kill the undead, we can help you get fixed up like us, yeah.  Stuck right into your flesh and bone and wired into your brain.  You understand?  If not, we'll ask the Great Grey to help clear it up. No: Ah, well.  I'll file that away, so to speak.  Maybe you could bring us a little more trade, and we'll fix you up as soon as possible.  The exodii won't turn away a willing convert.",
```
Comment (actual-nh) [inline data/json/npcs/exodii/exodii_merchant_Rubik.jsonbackup:199]:
```suggestion
    "//~": "English: Yes: Ah, now you're speaking my language.  We can fix you up good, if you're willing to pay us back.  I'll say it as clearly as I can.  If you help us kill the undead, we can help you get fixed up like us, yeah.  Stuck right into your flesh and bone and wired into your brain.  You understand?  If not, we'll ask the Great Grey to help clear it up. No: Ah, well.  I'll file that away, so to speak.  Maybe you could bring us a little more trade, and we'll fix you up as soon as possible.  The exodii won't turn away a willing convert.",
```
"Ah, well" sounds like Rubik is disappointed?

File: data/json/npcs/exodii/exodii_merchant_Rubik_stub.json [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+  {
+    "type": "mutation",
+    "id": "EXODII_BODY_1",
+    "flags": [ "Exodii_Body" ],
+    "name": { "str": "Exodii cyborg framework - style 1a" },
+    "points": 0,
+    "visibility": 4,
+    "ugliness": 4,
+    "mixed_effect": true,
+    "description": "Your brain and some of your organs are shielded in a nice tough bipedal walker frame.",
```
Comment (actual-nh) [inline data/json/npcs/exodii/exodii_merchant_Rubik_stub.json:49]: "tough" implies a bit of armor, at least to me. If this is meaning the HP adjustment, perhaps "resilient"?

File: data/json/npcs/exodii/exodii_merchant_itemlist.json [hunk 3]
```diff
@@ -0,0 +1,156 @@
+[
+  {
+    "type": "item_group",
+    "id": "NC_EXODII_TYPE_1_carried",
+    "//": "Weapon for rubik.",
+    "items": [ { "item": "pamd68", "ammo-item": "123ln", "charges": 10 } ]
+  },
+  {
+    "type": "item_group",
+    "id": "EXODII_SHOP_Tier0",
```
Comment (actual-nh) [inline data/json/npcs/exodii/exodii_merchant_itemlist.json:10]: Perhaps keep the capitalization more consistent?

**Review by actual-nh — COMMENTED** [review]
I like!

**Comment by tenmillimaster** [comment]
This is amazing, erk.

File: data/json/npcs/exodii/exodii_merchant_Rubik_stub.json (hunk 2 above)
Comment (I-am-Erk) [reply data/json/npcs/exodii/exodii_merchant_Rubik_stub.json]: It should have armour as well, I just haven't gotten around to it.

File: data/json/npcs/exodii/exodii_merchant_talk.json [hunk 4]
```diff
@@ -0,0 +1,704 @@
...
+      },
+      {
+        "text": "Maybe try that clarity-draught?",
+        "topic": "TALK_EXODII_MERCHANT_ExodizeTranslate",
+        "condition": {
+          "and": [
+            { "compare_int": [ { "u_val": "sold" }, { "const": 100 } ], "op": "<" },
+            { "u_compare_var": "exodii_trust", "type": "counter", "context": "exodii", "op": "<", "value": 5 }
+          ]
+        }
```
Comment (actual-nh) [inline data/json/npcs/exodii/exodii_merchant_talk.json:468]: I am still uncertain on why the "clarity-draught" would require low trust and not having sold much. Or am I misreading the code?

Comment (I-am-Erk) [reply data/json/npcs/exodii/exodii_merchant_talk.json:475]: Nah that's probably in the wrong spot. I'll take a look at what I meant to do.

**Comment by I-am-Erk** [comment]
I may add a few more links to get you to ask to become a cyborg before this is merged, but it's functional as is and safe to playtest

File: data/json/npcs/exodii/exodii_merchant_missions.json [hunk 5]
```diff
@@ -0,0 +1,25 @@
+[
+  {
+    "id": "RUBIK_ANUS_FETICK",
+    "type": "mission_definition",
+    "name": { "str": "Bring two anesthesia kits to Rubik" },
+    "goal": "MGOAL_FIND_ITEM",
+    "difficulty": 2,
+    "value": 240,
+    "item": "anesthetic_kit",
```
Comment (Venera3) [inline data/json/npcs/exodii/exodii_merchant_missions.json:9]: Anesthetic kit is the rubber hose+glass jar combo autolearned at healthcare 3, I think sending you for the `anus fetic` itself would make sense (and that makes you jump through a lot more hoops if you want to craft it).

Comment (I-am-Erk) [reply data/json/npcs/exodii/exodii_merchant_missions.json:9]: Forgot this was exclusive of charges, yeah I'll have to change that to specific drugs

File: data/json/npcs/exodii/exodii_merchant_talk.json [hunk 6]
```diff
@@ -0,0 +1,711 @@
...
+    ]
+  },
+  {
+    "id": "TALK_EXODII_MERCHANT_Talk_Intro3a",
+    "type": "talk_topic",
+    "dynamic_line": "Neverken.  Right and so, the enemy is new here, and us'n have a few good hours in the sun beforenext the jump.",
+    "responses": [
+      {
+        "text": "Oh, you mean you got here early in the cataclysm.  Why is that so good?",
+        "topic": "TALK_EXODII_MERCHANT_Talk_Intro3"
```
Comment (cainiaowu) [inline data/json/npcs/exodii/exodii_merchant_talk.json:325]: This seems like a infinite loop, maybe this one should be TALK_EXODII_MERCHANT_Talk_Intro4a ? 
TALK_EXODII_MERCHANT_Talk_Intro4a is not used at all.

Comment (I-am-Erk) [reply data/json/npcs/exodii/exodii_merchant_talk.json:325]: I did indeed. Thanks, will catch that on the next pass.

**Pull request merged**